App/Sagas/TextileSagas.ts

```diff
@@ -564,6 +565,120 @@ export function * refreshThreads () {
   }
 }
 
+export function * showImagePicker(action: ActionType<typeof UIActions.showImagePicker>) {
+  const { threadId } = action.payload
+
+  // Turn ImagePicker into a Promise
+  const pickerPromise = new Promise((resolve, reject) => {
```

Comment:
This code already exists in `CameraRoll.ts` if you want to use it. Just a simple function you can call that returns a `Promise<{ uri: string, data: string}>`

Reply:
ah, perf